variant_remapping_tools/reads_to_remapped_variants.py

```diff
@@ -18,6 +18,9 @@ def calculate_new_variant_definition(left_read, right_read, ref_fasta):
     Resolve the variant definition from the flanking region alignment and old variant definition
     TODO: Link to algorithm description once public
     """
+    # Flag to highlight low confidence in an event detected
+    low_confidence = False
```

Comment:
Can this flag could be named something more specific?  I imagine we might want to keep track of the different reasons variants are discarded, but maybe that's not the intent of this flag.

Reply:
Do you mean having multiple flags ? for the "novel reference allele" and the "change in reference allele length"  ?
The main purpose of the flag is to convey that we have low confidence in those events. Not sure what else to name it.

Reply:
Yeah multiple flags for these might make more sense.  Basically I'm being influenced by the OT pipeline, where we track the different reasons records fail to produce evidence strings, some of which we can fix in the future and some we can't.  I could see something similar happening with variant remapping in the future as we try to improve it.

Reply:
Good point by @apriltuesday here, I agree having high granularity on error modes is very useful.

One possibility is to swap `low_confidence` for something like `reason_for_failure`, and to make it a _string_ rather than a bool. That string could then be fed directly into the counter in the calling function.

Reply:
Ok that makes sense. I split the flag